tdrs-backend/tdpservice/parsers/case_consistency_validator.py

```diff
@@ -504,7 +519,8 @@ def __validate_t5_atd_and_ssi(self):
                     msg=(
                         f"{t5_model_name} People in states should not have a value "
                         f"of 1 for {self.__get_error_context('REC_AID_TOTALLY_DISABLED', schema)}."
-                    )
+                    ),
+                    deprecated=True
```

Comment:
confirmed the following no longer appears on the error report ✔️ 

`T5 People in states should not have a value of 1 for Item 19C (Received Disability Benefits: Permanently and Totally Disabled).`